### ISSUE TITLE
Unable to run the example

### ISSUE THREAD

**Issue by acro5piano** [Bear-03/vosk-rs#1]
Hi, thank you for the great library. I hope this issue does not bother you.

I've tried to run the example as documented in the source code but compilation failed. It seems that library cannot find the vosk dynamic library.

```sh
$ RURSTFLAGS=-L./lib/vosk \
       LD_LIBRARY_PATH=./lib/vosk \
       cargo run --example microphone lib/vosk-model 10

   Compiling vosk v0.1.0 (/home/kazuya/ghq/github.com/acro5piano/vosk-rs/vosk)
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/home/kaz............caf446e056b41091.13i6whe78lh07zg9.rcgu.o"

  # ...long text here...

  = note: /bin/ld: cannot find -lvosk: No such file or directory
          collect2: error: ld returned 1 exit status
```

I've placed vosk on `lib/vosk` directory so the directory structure is like this. Am I missing something?

```
~> tree -L 3
.
|-- Cargo.toml
|-- CHANGELOG.md
|-- examples
|   |-- microphone.rs
|   |-- read_wav.rs
|   `-- speaker_model.rs
|-- lib
|   |-- vosk
|   |   |-- libvosk.so
|   |   `-- vosk_api.h
|   `-- vosk-model
|       |-- am
|       |-- conf
|       |-- graph
|       |-- ivector
|       |-- README.txt
|       `-- rescore
|-- LICENSE -> ../LICENSE
|-- README.md -> ../README.md
`-- src
    |-- lib.rs
    |-- log.rs
    |-- models.rs
    `-- recognition
        |-- mod.rs
        `-- results.rs
```

I placed `build.rs` to tell linking directory to rust compiler - but no luck.

```rs
// build.rs

fn main() {
    println!("cargo:rustc-link-lib=./lib/vosk");
}
```
Thank you in advance.

**Comment by acro5piano** [comment]
I could successfully compile my app using this library (still not working well though, but this is off topic here).

**Comment by Bear-03** [comment]
> I could successfully compile my app using this library
No luck running it though, right? What OS are you using?

**Comment by acro5piano** [comment]
Thank you for the quick reply! I'm running it on Arch Linux.

**Comment by Bear-03** [comment]
I've noticed a typo in your command. Instead of `RURSTFLAGS=`, it should be `RUSTFLAGS=`

**Comment by Bear-03** [comment]
I fixed the typo in the README (commit 00d048c4b7516a073d1515c9ca67bd42639c9f96). Let met know if that fixes it and I'll bump the version.

**Comment by acro5piano** [comment]
Thank you for finding the typo! I'm sorry I don't well understand how Rust's build system works.

And I successfully run the example! I should have run the script on the top of repository (workspace root) instead of `/vosk` directory.

The succeeding command is:

```
RUSTFLAGS=-L./lib/vosk LD_LIBRARY_PATH=./lib/vosk cargo run --example microphone lib/vosk-model 10
```

So the final directory structure is like this:

```
.
|-- Cargo.lock
|-- Cargo.toml
|-- lib
|   |-- vosk
|   |   |-- libvosk.so
|   |   `-- vosk_api.h
|   `-- vosk-model
|       |-- am
|       |-- conf
|       |-- graph
|       |-- ivector
|       |-- README.txt
|       `-- rescore
|-- LICENSE
|-- README.md
|-- vosk
|   |-- Cargo.toml
|   |-- examples
|   |   |-- microphone.rs
|   |   |-- read_wav.rs
|   |   `-- speaker_model.rs
|   |-- LICENSE -> ../LICENSE
|   `-- src
|       |-- lib.rs
|       |-- log.rs
|       |-- models.rs
|       `-- recognition
`-- vosk-sys
    |-- Cargo.toml
    |-- LICENSE -> ../LICENSE
    `-- src
        `-- lib.rs
```
